src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
I'll just leave this here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1037. It didn't make it into https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/887.